manifests/b/buchen/portfolio/0.61.3/buchen.portfolio.locale.de-DE.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: buchen.portfolio
+PackageVersion: 0.61.3
+PackageLocale: de-DE
+Publisher: Andreas Buchen
+PackageName: Portfolio Performance
+License: EPL-1.0
+ShortDescription: Portfolio Performance calculates the overall perfromance of your investment portfolio.

```

Comment:
This should be translated as the PackageLocale used is `de-DE`.

Reply:
Thank you for the feedback. I've translated the ShortDescription using https://www.deepl.com/translator